sdks/python/apache_beam/testing/test_stream_service.py

```diff
@@ -50,9 +52,17 @@ def __init__(self, reader, endpoint=None, exception_handler=None):
       self._exception_handler = lambda _: False
 
   def start(self):
-    self._server.start()
+    # A server can only be started if never started and never stopped before.
+    if not self._server_started and not self._server_stopped:
+      self._server_started = True
+      self._server.start()
 
   def stop(self):
+    # A server can only be stopped if already started and never stopped before.
+    if not self._server_started or self._server_stopped:
```

Comment:
Write this in a format similar to start():

`if self._server_started and not self._server_stopped:`

or change start to similar format as

```
if self._server_started or self._server_stopped:
    return
```

Reply:
Thanks! Formatted it.